nestkernel/connection.h

```diff
@@ -120,6 +121,8 @@ class Connection
   // connections not used in primary connectors
   typedef SecondaryEvent EventType;
 
+  const static RegisterConnectionModelFlags flags = default_connection_model_flags;
```

Comment:
Can the definition of the defaults be moved from `nest.h` to this file as well? They should only be needed here, shouldn't they?

Reply:
Following up on this, couldn't this actually become (with some implicit suggestions for name changes)
```c++
constexpr ConnectionModelProperties properties = 
ConnectionModelProperties::SUPPORTS_HPC
  | ConnectionModelProperties::SUPPORTS_LBL
  | ConnectionModelProperties::IS_PRIMARY
  | ConnectionModelProperties::HAS_DELAY;
```
and then in the derived classes on could do
```c++
constexpr ConnectionModelProperties properties = ConnectionT::properties |           
             ConnectionModelProperties::REQUIRES_URBANCZIK_ARCHIVING;
```

And maybe rename `enumFlagSet()` to `has_property()` to focus on meaning and not implementation?